pep-0484.txt

```diff
@@ -610,6 +612,24 @@ the runtime class of the objects created by instantiating them doesn't
 record the distinction. This behavior is called "type erasure"; it is
 common practice in languages with generics (e.g. Java, TypeScript).
 
+Using generic classes (parameterized or not) to access attributes will result
+in both type check failure and runtime error. Outside the class definition
```

Comment:
> and runtime error.

Is it really a runtime error for examples below? I just tried it and all worked fine _at runtime_.